docs/sources/flow/upgrade-guide.md

```diff
@@ -56,6 +56,31 @@ pyroscope.scrape "default" {
 }
 ```
 
+### Breaking change: `service_name` label in is required in `pyroscope.scrape`
```

Comment:
Is there a word missing in this heading?

Reply:
thanks! fixed, made it 
```
`service_name` label in is required in `pyroscope.scrape`
```